python/pyspark/testing/utils.py

```diff
@@ -189,6 +190,39 @@ def search_jar(project_relative_path, sbt_jar_name_prefix, mvn_jar_name_prefix):
         return jars[0]
 
 
+def _context_diff(actual: List[str], expected: List[str], n: int = 3):
+    """
+    Modified from difflib context_diff API,
+    see original code here: https://github.com/python/cpython/blob/main/Lib/difflib.py#L1180
+    """
+    def red(s: str) -> str:
```

Comment:
I think we'll need a configuration to disable the colors. In some terminals, they do not work, and looks weird characters. One reference is `sbt -no-colors`